api/go.mod

```diff
@@ -6,7 +6,8 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/protobuf v1.5.2
 	github.com/google/go-cmp v0.5.7
-	github.com/gravitational/trace v1.1.17
+	github.com/gravitational/teleport v0.0.0
```

Comment:
Let's ditch the cyclic dependency. One of the reasons for api to exist is so that people using it don't have to pull the entirety of Teleport.

I would prefer to do the other go.mod changes in a separate PR too, so we can distinguish the updates from your changes  (unless there is something you do need from the newer modules).

Reply:
I had to include `github.com/gravitational/teleport` because I need the trait constants (ex `TraitLogins`, `TraitWindowsLogins`)
That triggered the update to the `trace` module

Should we move all the constants to a separate module? So that we can include them from here (api)

Reply:
I think it's OK moving the constants to api/, given that we need them there now.